L1Trigger/L1TTrackMatch/python/l1tTrackJets_cfi.py

```diff
@@ -1,17 +1,18 @@
 import FWCore.ParameterSet.Config as cms
 
+#prompt jet selection
 l1tTrackJets = cms.EDProducer('L1TrackJetProducer',
-	L1TrackInputTag= cms.InputTag("l1tTTTracksFromTrackletEmulation", "Level1TTTracks"),
-	L1PVertexCollection = cms.InputTag("l1tVertexProducer", "l1vertices"),
-	MaxDzTrackPV = cms.double( 1.0 ),
+        L1TrackInputTag= cms.InputTag("l1tTTTracksFromTrackletEmulation","Level1TTTracks"),
+        L1PVertexInputTag = cms.InputTag("l1tVertexFinderEmulator","l1verticesEmulation"),
+        MaxDzTrackPV = cms.double( 1.0 ), #max distance from PV;negative=no cut
 	trk_zMax = cms.double (15.) ,    # maximum track z
 	trk_ptMax = cms.double(200.),    # maximumum track pT before saturation [GeV]
-	trk_ptMin = cms.double(3.0),     # minimum track pt [GeV]
+	trk_ptMin = cms.double(2.0),     # minimum track pt [GeV]
```

Comment:
This was modified in "3.0" in #38337: please check and confirm that you want to revert here to the value previous to the merge of that PR

Reply:
Yes this is fine. It must agree with with the corresponding value of trackjetemulation cfg